community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/AllStoreHolder.java

```diff
@@ -262,7 +262,7 @@ void relationship( RelationshipRecord record, long reference, PageCursor pageCur
     @Override
     void property( PropertyRecord record, long reference, PageCursor pageCursor )
     {
-        properties.loadRecordByCursor( reference, record, RecordLoad.NORMAL, pageCursor );
+        properties.loadRecordByCursor( reference, record, RecordLoad.FORCE, pageCursor );
```

Comment:
Because ConsistentPropertyReadsIT, and it is what we currently do :) 

I'll try to motivate somehow, as you see I thought `NORMAL` was the more obvious choice...